catalog-info.yaml

```diff
@@ -37,6 +37,16 @@ apiVersion: backstage.io/v1alpha1
 kind: Component
 metadata:
   name: gcp-folder
+  annotations:
+    grafana/tag-selector: "Kubernetes"
+  tags:
+    - web
+    - frontend
+    - backstage
+  links:
+    - url: https://backstage.xxxxxx.xxx.xx
+      title: Backstage
+      icon: cloud
```

Comment:
Add this to a different resource. I was using this resource for gcp-folder.

Reply:
entry removed from catalog-info.
as of now configuration loading from entity, will see later if required to look-up the prj structure.